code/modules/detectivework/tools/swabs.dm

```diff
@@ -55,17 +55,17 @@
 				has_hand = 1
 		if(!has_hand)
 			to_chat(user, SPAN_WARNING("They don't have any hands."))
-			return
+			return FALSE
 		user.visible_message("[user] swabs [H]'s palm for a sample.")
 		sample_type = "GSR"
 		gsr = H.gunshot_residue
 	else
-		return
+		return FALSE
 
 	if(sample_type)
 		set_used(sample_type, H)
-		return
-	return 1
+		return TRUE
+	return TRUE
```

Comment:
Probably no need in `return TRUE` after `set_used(sample_type, H)`, since there is only another `return TRUE` after that.